docs/upgrade-notes/upgrade-rds-cluster-tf-postgres-13.md

```diff
@@ -0,0 +1,41 @@
+---
+id: upgrade-rds-cluster-tf-postgres-13
+title: Upgrade Database Cluster to PostgreSQL v13
+hide_title: false
+---
+
+Cumulus Core as of version > 18.1.0, and for the stable support release 16.1.x *only* for versions > v16.1.3 now supports and is tested against Aurora Postgres v13.   All users should update their datastores to this version as part of an upgrade process upon upgrading to release version 18.2.0 or 16.1.4.
+
+We recommend stopping all ingest rules if database downtime is required (e.g. you do not have a blue-green database solution or are using serverless V1) for the update  as any unavailability of the database may result in unexpected database write failures (resulting in records in the Dead Letter Archive), workflow failures or other unexpected failures.
+
+**It is also recommended that users test/evaluate the upgrade prior to performing it on their primary datastore in order to determine expected downtime and/or other related issues for their particular configuration.**
+
+## Users utilizing the `cumulus-rds-tf` module
```

Comment:
This is as opposed to users using their own custom configuration? There's probably not much to say but maybe another section to cover the rest of the users? Are there any relevant AWS docs we could point them to? I'm not certain what popular alternatives to our module would be so perhaps not.

Reply:
Right, at some point the means and methods they use to deploy, backup and maintain their primary datastore are up to the users.      This is just a reminder to not "YOLO" a database upgrade to 13 for any configuration without appropriate engineering rigor - it may be unnecessary....

Reply:
Yeah, makes sense. I think leave it to be explicit and no need to talk about alternatives.